lib/PlaidClient.js

```diff
@@ -182,6 +182,8 @@ Client.prototype.getAllTransactions =
 
     return wrapPromise(P.coroutine(function*() {
       var transactions = [];
+      var transactionsCount = 0;
+      var response = [];
```

Comment:
this should be an object, not an array, right @cconeill ?

Reply:
Updated to `response = {}` in 2cc1831.